apl-core/pom.xml

```diff
@@ -49,6 +49,16 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-joda</artifactId>
```

Comment:
Looks like redundant dep

Reply:
Library is used by DEX code, I just moved that into correct place.